docs/draw.io/data-model.drawio.xml

```diff
@@ -1 +1,141 @@
-<mxfile host="app.diagrams.net" modified="2022-06-06T01:58:54.042Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.0.0 Safari/537.36" etag="a1--w3fBj2wtx0qpCjRf" version="19.0.0" type="device"><diagram id="dun09WMIp-Qr-0l5YfAu" name="Page-1">7Vxbc9o4FP41zOzuTBhfuTyGJKTdJLvd0Jlt903Yiu3GWK4sAvTXr2RLvspgGsCkcTIT0LGu53z6ztHF6elXi/UtBqH7gGzo9zTFXvf0656mDUca/csEm0RgDI1E4GDPTkRqJph5PyAXKly69GwYFTIShHzihUWhhYIAWqQgAxijVTHbE/KLrYbAgRXBzAJ+VfqvZxM3kY5MJZN/gJ7jipZVhT9ZAJGZCyIX2GiVE+k3Pf0KI0SSb4v1FfSZ7oReknLTmqdpxzAMSJMC38Zf9JUyXY2IeX9nq+Hn26//XehJLS/AX/IB886SjdCAg9EyrDbG23+BmMC1zBRgLmrIRktRAtECEryh+Xgpg+tnUzT8KtO2JnTqFjXNrcwt7KQ1Z0qgX7ge9tCJulsnLlnQKq/Vnj5hCvAoXu7BHPqfUOQRDwX02RwRghY0A/A9hwksqjqIqcBnOSfAemaaDewr5CMcV6s/xT+5Si95WYJCKo0IRs8pDLVUkqtBUUbKVGFPBNpYwgaRC22eoE9CNozF2mEztu+haNj3qNGivuWjJc02qVp7K3jKEKiauj1jahJjDnzC9BCCoGDVwfclm4uTeWqaCyvR7CXNh535b5pp9jTaDyX/5fdYYZRZAnIRxfzFsmujcB0/ELXSbw77/PPj46XoAh1R0ovkUQVmVK1MnkNb0eABCmjOyZPn+yVRBXRlRC0822bNTFauR+AsBBZrc0URQWXx4FPAsJFxYtZGPF2A3HA6nR4DNcKHFAnCrBLEQAIp/ViIMtqnzHTA9ZypjseSaXY0pZgdZzZDv1GDga2cKTfm0ThzcHDOVHK0qeSYU1PqybPCnLcecZfzvkUR8p75szmCGvOnKYHX8fhTFl8l9mSaidlQKCeDAFUT+8mLJDAs4UjVZDhSc/BJWjxr+BRYyfFBFJ0JrkYlJyT1QlVgaccC1rB9x2yUddLUMR+Ny0edY24G/2ENBs7JMY/PwzHzZ74XwAsxcPZU6Y/rvLbo5xxnbDsACwaEYB6xjxnEVPfSfO/X1zcH5Xn6et2o2AnaDpzxJMLERQ4KgH+TSUsazfLcI0YcsWm/QUI2XM9gSVDR8E2NAAP7km0RZlCgkqnHhnitSI3EvUKEltiCW8atcpYlADtwm4F5hUwpW82LoQ+I91LclDy8ucw2zEW1izdfWPm+KZJfeXVx4npdSG32nGtnYGZtcFZ2Fv3OOZKPQUSAzxqnDr8MgmjlLXwQq85yPd++Bxu0ZOOlZaxnkZq4CHs/qBFAxsAACxPpSiHHjJXkBsgQVJq9qiHSvC+KNHDAkHmoT8KASiq6BxERXakNISr0X4ejfOzdPIasperyHs5AEivKNrbUbQE0b+2RxayBQ3W2T3NDSWtl1wB8qqQAEDhhRosqUEzH+Qp0Vg8laH42hXSV6d7yPar1O7jJyah5MbRn0MKQ5MTfPAw+IAqCMqKF0X34RPYIFWgEYnmBcx+XujZKaNVqg90YjnwRkSCeAJJLUyAJoJbQp16y3+YRQzq1d+NwU7TwLtyNj7ZFX91vmi3nkYW9sGOjlthIb4qKX56NtOq2QoGNnHh9k3efH+0SA11JCCtjplJNyTroMgxpNfr0/RBXygJviLiqmyuPMESzTWBRQiGwY64WmMtUOubi8KxuFxWYK8o5WcZZ74ZpRo0hdi5Mo+sV87yLfRSh+MOtm3nRT8iLj5q4+c1xaVaXuSFZ4fNSh5+qYpy5qXpbjAY6X3IKX2KqDcJg2Q7qL+9M9OqprWRRXgh9Q+y90D4V494VnLsIPVdW6kvsvx8PlE73V3kgGQ6P54Fkp6vdqT1r81Sn9nuzmaqPi/SiGhUUaRJ6OdoJvS47je5A9LZApA9aBlH9KXgHojcCovTuUFsgMqoB92lB9EcHoteCSFfMlkEke3umA9FZg8gs3ZDRZduFp7y1aMjeJ+jc2ZsCUbr6bw1Eg4q1Trs/GC+rKxqPIbp7fzC7ANTLX//JbgPVXACqteTOfUVxqJe/uPN48/0B3RHnr9Hd/GG++Wf24fODeFnp2PuP5Xt7hnBtJ9p/NLrV/ZsjoZRNhCdrPabuVvdvDkTGWN0JotN6snEbnuwAV1Bf5QEP68m2nnqeyRVUs+0FeEcWr16AG5LXC07qccy2XyXsFuCvB9HwdLs40hBfrRjrJA6n6ZJoP4f0847EPK8VkV6+kHq8GxnSYbby4tLPLoQPEl5ss/bO6EJcDzh9dLGt2zm/8PdjxaDYRYv5MqphzpxZDsB6uj7um6VIu8p7hoT3jP15jyazfyuVTIjsf3PpN/8D</diagram></mxfile>
\ No newline at end of file
+<mxfile host="app.diagrams.net" modified="2022-06-09T00:49:27.456Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.0.0 Safari/537.36" etag="3GRBDWr6KHYVjdFBL5xY" compressed="false" version="19.0.3" type="device">
```

Comment:
Hey @mboudreau I think you forgot to re-generate the jpg.

Reply:
whoops, good catch